OurUmbraco.Site/Views/Partials/Members/QuickMenu.cshtml

```diff
@@ -2,8 +2,17 @@
 @using OurUmbraco.Forum.Extensions
 @using OurUmbraco.Our.Extensions
 @using OurUmbraco.Our.Models
+
 @{
     var profile = MemberData;
+
+    var homePage = Umbraco.TypedContentAtRoot().FirstOrDefault();
+    var profilePage = homePage.Descendants("Textpage").Where(x => x.Name == "Profile").ToList().FirstOrDefault();
```

Comment:
`Descendants` is very expensive and this needs to be done for each logged in user each time. 

We're very happy just hardcoding this to `/member/profile/packages/` for now.

Reply:
@nul800sebastiaan wouldn't it make sense to introduce a constant here?

https://github.com/umbraco/OurUmbraco/blob/main/OurUmbraco/Constants.cs

Either for the route directly, or the key/GUID of the packages page?

Reply:
No key/GUID please. Can't guarantee the dev db has the same GUID.

I don't really think we need to write code to put a string in a single place for now. When the packages section gets overhauled it might be something we need.

Reply:
I have made that change 👍